src/jit/importer.cpp

```diff
@@ -1074,14 +1074,16 @@ GenTreePtr Compiler::impAssignStruct(GenTreePtr     dest,
 
 /*****************************************************************************/
 
-GenTreePtr Compiler::impAssignStructPtr(GenTreePtr      dest,
+GenTreePtr Compiler::impAssignStructPtr(GenTreePtr      destAddr,
                                         GenTreePtr      src,
                                         CORINFO_CLASS_HANDLE    structHnd,
                                         unsigned        curLevel,
                                         GenTreePtr    * pAfterStmt,  /* = NULL */
                                         BasicBlock    * block        /* = NULL */
                                        ) 
 {
+    GenTreePtr dest = nullptr;
```

Comment:
Nit: would you mind duplicating this local and sinking its declaration into the two scopes in which it is used?